### PR TITLE
GEOMESA-713 Fixing of attribute index job

### PR DESCRIPTION
**Pull request by elahrvivaz**
* Job can be run stand-alone without needing manual steps afterwards
* Fixed geoserver page that was failing to update the attributes after the job completes

Signed-off-by: Emilio Lahr-Vivaz <elahrvivaz@ccri.com>

### CODE REVIEW

**Comment by eclipsewebmaster** [comment]
:white_check_mark: 
#### Eclipse validation results:
All committers passed Eclipse CLA and Signed-off-by validation.

File: geomesa-core/src/main/scala/org/locationtech/geomesa/core/data/tables/AttributeTable.scala [hunk 1]
```diff
@@ -76,7 +76,7 @@ object AttributeTable extends GeoMesaTable with Logging {
                     rowIdPrefix: String): FeatureWriterFn = {
 
     val indexesOfIndexedAttributes = indexedAttributes.map { a => sft.indexOf(a.getName) }
-    val attributesToIdx = indexesOfIndexedAttributes.zip(indexedAttributes)
+    val attributesToIdx = indexedAttributes.zip(indexesOfIndexedAttributes)
```
Comment (jnh5y) [inline geomesa-core/src/main/scala/org/locationtech/geomesa/core/data/tables/AttributeTable.scala]: (picky) Does the order of the zipping matter?

File: geomesa-jobs/src/main/scala/org/locationtech/geomesa/jobs/index/AttributeIndexJob.scala [hunk 2]
```diff
@@ -16,187 +16,94 @@
...
+    AttributeTable.getAttributeIndexMutations(feature, r.ive, r.fe, r.attrs, r.visibilities, r.prefix)
   }
 
-  def runJob(conf: Configuration, params: Map[String, String], feature: String, attributes: Seq[String]) = {
-
-    if (attributes.isEmpty) {
-      throw new IllegalArgumentException("No attributes specified")
-    }
-
+  override def afterJobTasks() = {
```
Comment (jnh5y) [inline geomesa-jobs/src/main/scala/org/locationtech/geomesa/jobs/index/AttributeIndexJob.scala]: Ah nice!

File: geomesa-core/src/main/scala/org/locationtech/geomesa/core/data/tables/AttributeTable.scala (hunk 1 above)
Comment (elahrvivaz) [inline geomesa-core/src/main/scala/org/locationtech/geomesa/core/data/tables/AttributeTable.scala]: yeah, i flipped it b/c I realized that there are a lot of places where I had to zipWithIndex and then reverse the result :P

**Comment by jnh5y** [comment]
Committed as d63d975e89e2612e226cfab9d777dd277308dae6 on 1.5.

**Pull request closed without merging**